notebooks/quickstart_tutorial.ipynb

```diff
@@ -334,15 +334,19 @@
    "source": [
     "from cellarium.cas._io import suppress_stderr\n",
     "from cellarium.cas.visualization import CASCircularTreePlotUMAPDashApp\n",
+    "from cellarium.cas.postprocessing import insert_cas_ontology_aware_response_into_adata\n",
+    "\n",
+    "# Insert the CAS ontology-aware cell type query response into the AnnData object for the visualization application\n",
+    "insert_cas_ontology_aware_response_into_adata(cas_ontology_aware_response, adata)\n",
     "\n",
     "DASH_SERVER_PORT = 8050\n",
     "\n",
     "with suppress_stderr():\n",
     "    CASCircularTreePlotUMAPDashApp(\n",
-    "        adata,  # the AnnData file\n",
-    "        cas_ontology_aware_response,  # CAS response\n",
+    "        adata=adata,  # the AnnData file\n",
+    "        root_node=\"CL_0000255\",  # set to CL root node to \"eukaryotic cell\"\n",
     "        cluster_label_obs_column=\"cluster_label\",  # (optional) The .obs column name containing cluster labels \n",
-    "    ).run(port=DASH_SERVER_PORT, debug=False, jupyter_width=\"100%\")"
+    "    ).run(port=DASH_SERVER_PORT, debug=False, jupyter_width=\"1000\")"
```

Comment:
I'd leave this as 100%.  This shouldn't be screen specific (especially your gigantic monitor :) )

Reply:
Done.